src/themes/light-theme.js

```diff
@@ -5,245 +5,244 @@ This source code is licensed under the MIT license found in the
 LICENSE file in the root directory of this source tree.
 */
 // @flow
+
 // color constants
-const primary50 = '#EDF3FE';
-const primary100 = '#D2E0FC';
-const primary200 = '#9CBCF8';
-const primary300 = '#548BF4';
-const primary400 = '#276EF1';
-const primary500 = '#174EB6';
-const primary600 = '#123D90';
-const primary700 = '#0C2960';
+const defaultTheme = {
+  primary50: '#EDF3FE',
+  primary100: '#D2E0FC',
+  primary200: '#9CBCF8',
+  primary300: '#548BF4',
+  primary400: '#276EF1',
+  primary500: '#174EB6',
+  primary600: '#123D90',
+  primary700: '#0C2960',
+
+  negative50: '#FDF0EF',
+  negative100: '#FADBD7',
+  negative200: '#F4AFA7',
+  negative300: '#EB7567',
+  negative400: '#E54937',
+  negative500: '#AE372A',
+  negative600: '#892C21',
+  negative700: '#5C1D16',
 
-const negative50 = '#FDF0EF';
-const negative100 = '#FADBD7';
-const negative200 = '#F4AFA7';
-const negative300 = '#EB7567';
-const negative400 = '#E54937';
-const negative500 = '#AE372A';
-const negative600 = '#892C21';
-const negative700 = '#5C1D16';
+  warning50: '#FEF3EC',
+  warning100: '#FBE2CF',
+  warning200: '#F6BA8B',
+  warning300: '#F19248',
+  warning400: '#ED6F0E',
+  warning500: '#B4540B',
+  warning600: '#8E4308',
+  warning700: '#5F2C06',
 
-const warning50 = '#FEF3EC';
-const warning100 = '#FBE2CF';
-const warning200 = '#F6BA8B';
-const warning300 = '#F19248';
-const warning400 = '#ED6F0E';
-const warning500 = '#B4540B';
-const warning600 = '#8E4308';
-const warning700 = '#5F2C06';
+  positive50: '#EBF8F2',
+  positive100: '#CDEDDE',
+  positive200: '#88D3B0',
+  positive300: '#43B982',
+  positive400: '#07A35A',
+  positive500: '#057C44',
+  positive600: '#046236',
+  positive700: '#034124',
 
-const positive50 = '#EBF8F2';
-const positive100 = '#CDEDDE';
-const positive200 = '#88D3B0';
-const positive300 = '#43B982';
-const positive400 = '#07A35A';
-const positive500 = '#057C44';
-const positive600 = '#046236';
-const positive700 = '#034124';
+  mono100: '#FFFFFF',
+  mono200: '#F7F7F7',
+  mono300: '#F0F0F0',
+  mono400: '#E5E5E5',
+  mono500: '#CCCCCC',
+  mono600: '#B3B3B3',
+  mono700: '#999999',
+  mono800: '#666666',
+  mono900: '#333333',
+  mono1000: '#000000',
 
-const mono100 = '#FFFFFF';
-const mono200 = '#F7F7F7';
-const mono300 = '#F0F0F0';
-const mono400 = '#E5E5E5';
-const mono500 = '#CCCCCC';
-const mono600 = '#B3B3B3';
-const mono700 = '#999999';
-const mono800 = '#666666';
-const mono900 = '#333333';
-const mono1000 = '#000000';
+  primaryFontFamily: '"Helvetica Neue", arial, sans-serif',
+};
 
-export default function(
-  {
-    primaryFontFamily,
-  }: {
-    primaryFontFamily: string,
-  } = {
-    primaryFontFamily: '"Helvetica Neue", arial, sans-serif',
-  },
-) {
+// $FlowFixMe
+export default function(overrides) {
+  const theme = Object.assign({}, defaultTheme, overrides);
```

Comment:
This should probably be a deep merge

Reply:
Why is that? we only a single level of properties

Reply:
Typography is two level, i.e. I think this might end up overriding all of `typography`:
```js
Object.assign({}, defaultTheme, {
  typography: {
   myNewFontStyles: {...}
  }
}
```